### ISSUE TITLE
CLI export to SVG doesn't render LaTeX (but exports correctly in GUI)

### ISSUE THREAD

**Issue by jeremy-dolan** [jgraph/drawio-desktop#1874]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

---

Math between \\( \\) will render correctly when using SVG export in GUI, but when exported to SVG using CLI the 'source' LaTeX markup is shown.

This appears to have regressed, as the same issue was reported two years ago as issue #223, and then apparently fixed.

Tested on MacOSX Draw.io 24.7.8 (the latest desktop version)

**To Reproduce**
/Applications/draw.io.app/Contents/MacOS/draw.io -x -f svg -o figure.svg input.drawio

**Comment by JRodez** [comment]
Same on NixOS with 24.7.8

**Comment by kestr31** [comment]
- Image with Issue:

![image](https://github.com/user-attachments/assets/63adbc18-f6d4-4d9d-932a-f59931423a7f)

- Intended Image

![probabilitySpace](https://github.com/user-attachments/assets/85c21d8f-06f8-4856-b76c-88df81e81c69)

- I am experiencing the same issue. Kubuntu 24.04.1, DrawIO v24.7.17.
- Exported with following CLI command:

```bash
drawio -x -f svg --scale 2 "$filename.drawio" -o "${filename}.svg"
```

**Comment by jeremy-dolan** [comment]
Verified with newly released desktop client 25.0.2 that math is now being rendered with command line export (as mentioned in release notes). Two differences between GUI and CLI export continue to block automation for me:

1. Math exported via CLI is not anti-aliased. I don't know enough about SVG format to debug the file differences further. I've tested with default export settings as well as --scale 1.25, and it lacks the anti-aliasing of the GUI export in both cases. Screenshots of CLI and GUI exports, respectively (note the 1/sqrt(d_head) and the numbers in the bottom row of boxes, which are \(math\) text, as opposed to the top numbers which are regular text, and anti-aliased correctly):
<img width="173" alt="CLI-export" src="https://github.com/user-attachments/assets/e10f0714-2f7a-476d-9572-c92b03c07a48">
<img width="211" alt="GUI-export" src="https://github.com/user-attachments/assets/45b8fe58-2310-4521-b947-a3e78870cbe7">

2. SVG export via CLI doesn't honor --border. Perhaps same root cause at the issue about JPEG export not honoring --border: #1835

I'm happy to open a new issue for the anti-aliasing if you prefer to close this one, since math is now rendering--if poorly. Thanks!
